reference/configuration.html.markerb

```diff
@@ -184,6 +184,16 @@ The `release_command` value replaces `CMD` in the temporary Machine. This is use
 The temporary Machine has full access to the network, environment variables and secrets, but *not* to persistent volumes. Changes made to the file system on the temporary Machine will not be retained or deployed. The building/compiling of your project should be done in your Dockerfile. If you need to modify persistent volumes or configure your application, consider making use of `CMD` or `ENTRYPOINT` in your Dockerfile, or of a [process group command](#the-processes-section).
 
 The temporary Machine inherits the size from the largest Machine in the default process group of the app as of flyctl v0.0.508 (they also default larger on empty/new apps, using the Machine `shared-cpu-2x` preset). The default process group is defined as either the `app` process, or the first process group in alphabetical order (if an `app` process is not present).
+It is possible to explicitly set the Machine size by setting `[deploy.release_command_vm]`  like in:
```

Comment:
"It is also possible to explicitly define the Machine size..." mainly to better chain from the previous sentences and avoid a bit of repetition?